iceoryx2-ffi/cxx/README.md

```diff
@@ -57,15 +60,15 @@ The C++ bindings can use the existing Rust artifacts via
 projects. This are the steps:
 
 ```bash
-cmake -S . -B target/ffi/build -DCMAKE_INSTALL_PREFIX=target/ffi/install -DCMAKE_PREFIX_PATH="$( pwd )/target/iceoryx/install" -DRUST_BUILD_ARTIFACT_PATH="$( pwd )/target/release"
-cmake --build target/ffi/build
-cmake --install target/ffi/build
+cmake -S . -B target/ffi/c-cxx/build -DCMAKE_INSTALL_PREFIX=target/ffi/cxx/install -DCMAKE_PREFIX_PATH="$( pwd )/target/iceoryx/install" -DRUST_BUILD_ARTIFACT_PATH="$( pwd )/target/release"
+cmake --build target/ffi/c-cxx/build
+cmake --install target/ffi/c-cxx/build
```

Comment:
I need to update this for the integrators to build the C++ bindings based on the previously build C bindings, similar to `iceoryx_platform` and `iceoryx_hoofs`

Reply:
@elBoberido 

Still a work-in-progress but I am experimenting with completely decoupling both the C and C++ builds, and enforcing the user to build the C bindings before the C++ bindings: https://github.com/eclipse-iceoryx/iceoryx2/pull/926/commits/b568afa5ea85ce7c149f033959718f76f4f6e058#diff-d6dfb65f306a09b02050c0ed0d1024e4323cc1928f3a77ff80ffe23e2ae801fd

Currently is only an experiment for discussion - need to clarify if such a change will cause problems with users who depend on the old setup.